srcpkgs/TheForceEngine/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'TheForceEngine'
+pkgname=TheForceEngine
+version=1.09.410
+revision=1
+archs="x86_64"
+build_style=cmake
+configure_args="-DCMAKE_BUILD_TYPE=Release"
```

Comment:
What's wrong with our defaults for CMAKE_BUILD_TYPE?

Reply:
I removed it and it still works so I'll update for that.

https://github.com/void-linux/void-packages/blob/cbd64671f6059fe5553828532eae10df2cae8158/common/build-style/cmake.sh#L56 just curious does this mean our CMAKE_BUILD_TYPE is usually `None`? Or does it get set to something else later?